cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -723,6 +725,41 @@ public void testRestApiPortNetworkPolicy() {
         assertThat(rules.contains(clusterOperatorPeer), is(true));
     }
 
+
+    @Test
+    public void testGoalsCheck() {
+
+        String customGoals = "com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal," +
```

Comment:
I wonder, if we are re-using these strings, might we want to set each as a constant in the CruiseControl class to avoid string repetition?

Reply:
Did as you suggested and pointed all goal refs at these constants. Also made the default goal configs actual lists so we can programatically use them if we need to at a later date.